bookkeeper-server/src/test/java/org/apache/bookkeeper/client/GenericEnsemblePlacementPolicyTest.java

```diff
@@ -132,9 +132,9 @@ public void testReplaceBookie() throws Exception {
                     lh.addEntry(value);
                 }
             }
-            assertEquals(2, customMetadataOnNewEnsembleStack.size());
```

Comment:
yes this is a "change", but it is not possible to do in another way, because we need to call the 'legacy' method by default  and the class in the the does not override it.
I have preferred to change the assertion instead of changing the test policy, the fact that the legacy method is called is covered by the new test class